### ISSUE TITLE
kubernetes deployment or helm chart?

### ISSUE THREAD

**Issue by paulmorabito** [hooram/ownphotos-backend#92]
Hi,

Do you have any plans for creating kubernetes sample yaml files or a helm chart?

Thanks

**Comment by guysoft** [comment]
I never really did that. Would be happy to help you figure out what needs
to set up.

**Comment by paulmorabito** [comment]
I'll give it a try when I get some time and let you know if I have
questions. Thanks.

**Comment by chrispyduck** [comment]
Did you end up writing one, @paulmorabito? I'm looking at doing it but don't want to duplicate effort.

**Comment by derneuere** [comment]
Hey, ownphotos is no longer maintained. Here is a fork of ownphotos that is maintained: https://github.com/LibrePhotos/librephotos

I think on our discord a couple of people in the dev channel are working on a helm chart

**Comment by paulmorabito** [comment]
No, sorry. I had a few early versions but didn't get far enough along with
him before getting distracted on other things.

Good luck with it.
